lab-steve/test/auth-router-test.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+//load test environment
+require('dotenv').config({path: './test/.env'});
+const API_URL = `http://localhost:${process.env.PORT}`;
+
+//npm modules
+const request = require('superagent');
+const expect = require('expect');
+
+//app modules
+const server = require('../lib/server.js');
+const cleanDB = require('./lib/clean-db.js');
+const mockUser = require('./lib/mock-user.js');
+
+describe('testing auth router', () => {
+  before(server.start);
+  after(server.stop);
+  afterEach(cleanDB);
+
+  describe('testing 404', () => {
+    it('should respond with a 404 status code if invalid route', () => {
+      return request.post(`${API_URL}/bad/route`)
```

Comment:
nitpicky, but the indentation and nested returns/statements could be cleaned up here. how would you set up a mock of this data?